charts/intel-gaudi-resource-driver/templates/device-class.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: resource.k8s.io/v1alpha3
+apiVersion: resource.k8s.io/v1beta1
```

Comment:
v1beta1 is a part of K8s 1.32 which we support with Gaudi v0.3.0 release, if you change API version to beta, please bump up the image tag version to v0.3.0, it's out already, so it's okay.